.buildkite/hooks/prepare-release.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
Just a refactor so specific things for the release pipelines are not in the same script